docs/device-management-test.md

```diff
@@ -107,7 +107,7 @@ Occasionally, if the test failed during a previous attempt, the SMCC Greentea te
 
 #### Device identity is inconsistent
 
-If your device ID in Pelion Device Management is inconsistent over a device reset, it could be because it is failing to open the credentials on the storage held in the Enhanced Secure File System. Typically, this is because the device cannot access the Root of Trust stored in SOTP.
+If your device ID in Device Management is inconsistent over a device reset, it could be because it is failing to open the credentials on the storage held in the Enhanced Secure File System. Typically, this is because the device cannot access the Root of Trust stored in SOTP.
```

Comment:
"Over"?

Reply:
Maybe "... is inconsistent when your device resets, it ..."?

Reply:
Sure.